src/function_run_result.rs

```diff
@@ -2,6 +2,8 @@ use colored::Colorize;
 use serde::{Deserialize, Serialize};
 use std::fmt;
 
+const FUNCTION_LOG_LIMIT: usize = 20_000;
+
```

Comment:
Logs are at most 1kB (1,000 bytes), [from the docs](https://shopify.dev/docs/apps/functions#:~:text=Function%20logs%20to%20STDERR%20are%20truncated%20after%201%20kB.%20Some%20Wasm%20toolchains%20might%20crash%20if%20STDERR%20fails%20to%20write%20full%20logs.):
> Function logs to STDERR are truncated after 1 kB. Some Wasm toolchains might crash if STDERR fails to write full logs.